pkg/oauth/apis/oauth/validation/validation.go

```diff
@@ -45,6 +46,10 @@ func ValidateTokenName(name string, prefix bool) []string {
 	if len(name) < MinTokenLength {
 		return []string{fmt.Sprintf("must be at least %d characters long", MinTokenLength)}
 	}
+
+	if !strings.HasPrefix(name, sha256Prefix) {
+		return []string{fmt.Sprintf("only the new format of tokens (prefixed with sha256~) is allowed")}
+	}
```

Comment:
will this update of existing tokens?

Reply:
Current validation does not allow updating anything else but the `inactivityTimeoutSeconds`, so existing tokens will not be updated. The `inactivityTimeoutSeconds` of existing tokens will no longer be possible to be updated, so those will just time out.

Other, long-lived pre-existing tokens may still survive.

Reply:
ack

Reply:
btw: the newest changes make the old tokens disfunctional for authn